styles/style.css

```diff
@@ -0,0 +1,112 @@
+
+/* roboto-regular - latin_cyrillic */
+@font-face {
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 400;
+    src: local(''),
+    url('../fonts/roboto-v29-latin_cyrillic-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
+    url('../fonts/roboto-v29-latin_cyrillic-regular.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+/* roboto-500 - latin_cyrillic */
+@font-face {
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 500;
+    src: local(''),
+    url('../fonts/roboto-v29-latin_cyrillic-500.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
+    url('../fonts/roboto-v29-latin_cyrillic-500.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+
+html {
+    box-sizing: border-box;
+}
+
+* {
+    margin: 0;
+}
+
+.section {
+    margin: 0 auto;
+}
+
+h2 {
+    margin: 0 auto;
+    width: 306px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 40px;
+    line-height: 47px;
+    color: #383838;
+    padding-top: 120px;
+    text-align: center;
+}
+
+.line {
+    display: block;
+    margin: 0 auto;
+    padding-top: 20px;
+}
+
+.cards {
+    list-style: none;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+    padding: 80px 104px 120px;
+    align-items: center;
+}
+.cards li {
+    min-width: 233px;
+    margin: 0 16px;
+    text-align: center;
+}
+.cards img {
+    align-items: center;
+    height: 85px;
+    width: 85px;
+}
+.cards li p {
+    padding-top: 48px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 28px;
+}
+
+.footer {
+    max-height: 630px;
+    background-color: #10367C;
+    font-family: Roboto;
+    color: #ffffff;
+    display: flex;
+    padding-top: 120px;
+    padding-bottom: 80px;
+    justify-content: space-around;
+
+}
+
+.footer ul {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    line-height: 23px;
+    text-align: left;
+    list-style: none;
+}
+
+.footer ul li:first-child {
+    font-weight: 500;
+}
+
+.footer ul li {
+    padding-bottom: 24px;
+}
+
+.copyright {
+    padding: 180px 0 120px 0;
+    font-size: 14px;
+    line-height: 14px;
```

Comment:
Check margins and paddings
<img width="1437" alt="Screenshot 2022-03-11 at 19 45 31" src="https://user-images.githubusercontent.com/25488411/157910916-d5030fb4-fb21-4134-830f-c58bb46ab81a.png">